mmcv/cnn/rfsearch/operator.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+
+import numpy as np
+import torch
+import torch.nn as nn
+from mmengine.logging import MMLogger
+from mmengine.model import BaseModule
+from torch import Tensor
+
+from .utils import expand_rates, get_single_padding
+
+logger = MMLogger.get_current_instance()
```

Comment:
We should not get `logger` here, otherwise, this logger will not be the same logger in `Runner`